app/frontend/models/template-version.ts

```diff
@@ -94,6 +100,40 @@ export const TemplateVersionModel = types
 
       return response
     }),
+    copyJurisdictionTemplateVersionElectives: flow(function* (jurisdictionId: string) {
+      const response = yield* toGenerator(
+        self.environment.api.copyJurisdictionTemplateVersionCustomization(self.id, jurisdictionId, true, false, true)
+      )
+      if (!response.ok) {
+        return response
```

Comment:
just verifying if the whole response object was intended to be returned or just true/ false